src/router.js

```diff
@@ -15,6 +16,7 @@ const Router = () => (
       <Route exact path="/pinned-posts" component={PinnedPosts} />
       <Route exact path="/profile" component={Profile} />
       <Route exact path="/organization" component={Organization} />
+      <Route component={NotFound} />
```

Comment:
path is missing for not found route that is 404

Reply:
Sir no specific path is required there. The component 'NotFound' will render at every path not already specified above it. Therefore showing a 404 error for every route which is not specified in the app, and the user tries to access it.

Reply:
so usually, I have seen route as * for 404, please let me know if its not needed so that I can merge.